stable/mlrun/values.yaml

```
@@ -162,6 +162,10 @@ httpDB:
 defaultDockerRegistryURL: ""
 defaultDockerRegistrySecretName: ""
 
+# Fill this in order to enable working with Nuclio
+nuclioAPIURL: 'unknown'
+nuclioUIURL: 'unknown'
```

```suggestion
nuclioUIURL: 'disabled'
```
I think this value would be self-explanatory, because we'll have to ask UI to disable, for example, showing nuclio links, when there is no nuclio.

---

changed